shared/testutil/block.go

```diff
@@ -511,36 +511,8 @@ func HydrateV1BeaconBlockBody(b *v1.BeaconBlockBody) *v1.BeaconBlockBody {
 	return b
 }
 
-func NewPandoraBlockWithCorrectTime(
-	slot types.Slot,
-	proposerIndex uint64,
-	genesisTime uint64,
-) (header *gethTypes.Header, hash common.Hash, extraData *pandora.ExtraData) {
-	headerTime, err := helpers.SlotToTime(genesisTime, slot)
-
-	if err != nil {
-		return nil, gethTypes.EmptyRootHash, nil
-	}
-
-	headerTimeUnix := uint64(headerTime.Unix()) + uint64((time.Millisecond * 50).Seconds())
-
-	header, hash, extraData = NewPandoraBlock(slot, proposerIndex)
-
-	if nil == header {
-		return
-	}
-
-	header.Time = headerTimeUnix
-	hash = sealHash(header)
-
-	return
-}
-
 // getDummyBlock method creates a brand new block with extraData
-func NewPandoraBlock(
-	slot types.Slot,
-	proposerIndex uint64,
-) (*gethTypes.Header, common.Hash, *pandora.ExtraData) {
+func NewPandoraBlock(slot types.Slot, proposerIndex uint64) (*gethTypes.Header, *pandora.ExtraData) {
```

Comment:
why you use proposerIndex?

Reply:
@atif-konasl I think there is a `typo` also: `getDummyBlock` vs `NewPandoraBlock`